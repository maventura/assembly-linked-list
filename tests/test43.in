--- test 0 de 50
crear: tipo 1. valor -22.164166
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor 58.097915
concatenar: nodo 0
imprimir: antes   de filtrar:
filtrar: tipo 1
imprimir: despues de filtrar:
ahijar: nodo 1 en columna 16
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor -18.042957
crear: tipo 1. valor 44.398885
crear: tipo 1. valor 72.802789
crear: tipo 1. valor -20.896375
concatenar: nodo 3
crear: tipo 1. valor 44.999912
concatenar: nodo 4
concatenar: nodo 5
ahijar: nodo 2 en columna 21
crear: tipo 1. valor 7.574415
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor 68.531216
crear: tipo 1. valor 34.891568
crear: tipo 1. valor 33.823805
ahijar: nodo 8 en columna 10
ahijar: nodo 6 en columna 16
concatenar: nodo 10
crear: tipo 1. valor -20.532204
crear: tipo 1. valor 29.058479
concatenar: nodo 12
crear: tipo 1. valor 37.631634
crear: tipo 1. valor 10.625472
crear: tipo 1. valor 29.082299
concatenar: nodo 11
crear: tipo 1. valor 34.581087
concatenar: nodo 15
crear: tipo 1. valor -0.113546
concatenar: nodo 7
crear: tipo 1. valor -19.042904
ahijar: nodo 18 en columna 17
ahijar: nodo 14 en columna 46
concatenar: nodo 13
crear: tipo 1. valor 21.346565
concatenar: nodo 19
crear: tipo 1. valor 22.488795
crear: tipo 1. valor -0.118326
concatenar: nodo 16
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor -4.652737
crear: tipo 1. valor -8.751291
ahijar: nodo 22 en columna 5
concatenar: nodo 21
crear: tipo 1. valor 37.782087
crear: tipo 1. valor 23.956097
crear: tipo 1. valor -12.788591
concatenar: nodo 20
crear: tipo 1. valor 60.606395
ahijar: nodo 24 en columna 15
crear: tipo 1. valor -1.549073
imprimir: antes   de filtrar:
filtrar: tipo 1
imprimir: despues de filtrar:
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor 26.858057
concatenar: nodo 29
crear: tipo 1. valor 29.053990
ahijar: nodo 26 en columna 46
crear: tipo 1. valor 46.952414
concatenar: nodo 27
crear: tipo 1. valor 62.370257
concatenar: nodo 28
concatenar: nodo 17
crear: tipo 1. valor -5.300846
crear: tipo 1. valor 52.534325
crear: tipo 1. valor 56.484377
crear: tipo 1. valor 54.111544
concatenar: nodo 34
concatenar: nodo 36
ahijar: nodo 32 en columna 25
crear: tipo 1. valor -20.499676
ahijar: nodo 35 en columna 27
crear: tipo 1. valor 50.841434
crear: tipo 1. valor 34.007023
crear: tipo 1. valor 28.786928
crear: tipo 1. valor 67.625187
crear: tipo 1. valor 71.529624
concatenar: nodo 37
concatenar: nodo 9
concatenar: nodo 25
crear: tipo 1. valor 47.677041
crear: tipo 1. valor 19.094731
crear: tipo 1. valor 74.909973
concatenar: nodo 23
crear: tipo 1. valor 0.599361
crear: tipo 1. valor 7.170891
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
concatenar: nodo 42
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
ahijar: nodo 47 en columna 34
concatenar: nodo 40
crear: tipo 1. valor 53.318290
concatenar: nodo 38
crear: tipo 1. valor 37.466873
imprimir: antes   de filtrar:
filtrar: tipo 1
imprimir: despues de filtrar:
concatenar: nodo 31
concatenar: nodo 48
ahijar: nodo 43 en columna 34
concatenar: nodo 30
concatenar: nodo 45
concatenar: nodo 44
concatenar: nodo 39
concatenar: nodo 41
ahijar: nodo 49 en columna 21
concatenar: nodo 33
ahijar: nodo 46 en columna 9
borrar
--- test 1 de 50
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
crear: tipo 2. valor e1a
crear: tipo 2. valor sucrtj
ahijar: nodo 0 en columna 18
crear: tipo 2. valor dvidqgc8u
crear: tipo 2. valor 40r5t
concatenar: nodo 3
concatenar: nodo 1
crear: tipo 2. valor 6bl
crear: tipo 2. valor c2
crear: tipo 2. valor b3j
crear: tipo 2. valor 8fiawp72
concatenar: nodo 6
concatenar: nodo 4
imprimir: antes   de filtrar:
filtrar: tipo 2
imprimir: despues de filtrar:
crear: tipo 2. valor w
ahijar: nodo 7 en columna 36
crear: tipo 2. valor kk
crear: tipo 2. valor ka4mk98k8
concatenar: nodo 5
concatenar: nodo 9
crear: tipo 2. valor s13f
ahijar: nodo 11 en columna 25
ahijar: nodo 10 en columna 22
crear: tipo 2. valor v4w8amil
crear: tipo 2. valor apbcxw
crear: tipo 2. valor 5yjw
concatenar: nodo 13
crear: tipo 2. valor gs1a
crear: tipo 2. valor 57yr
ahijar: nodo 15 en columna 15
crear: tipo 2. valor i982yg3
concatenar: nodo 14
concatenar: nodo 16
crear: tipo 2. valor o0d2i3
crear: tipo 2. valor 4
concatenar: nodo 18
concatenar: nodo 12
crear: tipo 2. valor n5wf
ahijar: nodo 2 en columna 7
crear: tipo 2. valor nhf0
ahijar: nodo 21 en columna 45
ahijar: nodo 20 en columna 5
crear: tipo 2. valor cs
crear: tipo 2. valor bdlok8
concatenar: nodo 22
concatenar: nodo 8
concatenar: nodo 17
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
crear: tipo 2. valor wh6
concatenar: nodo 24
concatenar: nodo 19
crear: tipo 2. valor nifw3q
imprimir: antes   de filtrar:
filtrar: tipo 2
imprimir: despues de filtrar:
crear: tipo 2. valor d6c
crear: tipo 2. valor gg5lwfl4
ahijar: nodo 25 en columna 34
crear: tipo 2. valor fg
crear: tipo 2. valor 75fvq70m
crear: tipo 2. valor f6k
crear: tipo 2. valor fie
concatenar: nodo 27
concatenar: nodo 28
crear: tipo 2. valor r5dgekld
crear: tipo 2. valor d65h
concatenar: nodo 30
concatenar: nodo 33
crear: tipo 2. valor 66
concatenar: nodo 34
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
ahijar: nodo 26 en columna 41
crear: tipo 2. valor x
concatenar: nodo 32
concatenar: nodo 35
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
concatenar: nodo 29
crear: tipo 2. valor j4i3ilx9m
crear: tipo 2. valor x9ut8i
crear: tipo 2. valor bbpgppo
crear: tipo 2. valor hiaa3ya2
concatenar: nodo 38
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
crear: tipo 2. valor ltmv20
crear: tipo 2. valor wk
crear: tipo 2. valor maqfhyr
crear: tipo 2. valor u860k5r8o
crear: tipo 2. valor tfox
concatenar: nodo 41
crear: tipo 2. valor khjpokkl
crear: tipo 2. valor p09cu
imprimir: antes   de filtrar:
filtrar: tipo 2
imprimir: despues de filtrar:
concatenar: nodo 37
crear: tipo 2. valor gto
crear: tipo 2. valor 93
concatenar: nodo 23
crear: tipo 2. valor 6lmgw4aivc
ahijar: nodo 45 en columna 21
ahijar: nodo 42 en columna 49
concatenar: nodo 46
concatenar: nodo 49
concatenar: nodo 31
ahijar: nodo 44 en columna 36
concatenar: nodo 36
concatenar: nodo 40
concatenar: nodo 48
concatenar: nodo 39
ahijar: nodo 43 en columna 28
ahijar: nodo 47 en columna 49
borrar
--- test 2 de 50
crear: tipo 0. valor -2780
ahijar: nodo 0 en columna 29
crear: tipo 0. valor -3691
crear: tipo 0. valor 159
crear: tipo 0. valor 3552
crear: tipo 0. valor -4010
crear: tipo 0. valor -2414
crear: tipo 0. valor -417
concatenar: nodo 6
concatenar: nodo 1
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 5
ahijar: nodo 2 en columna 49
concatenar: nodo 3
concatenar: nodo 4
crear: tipo 0. valor 3601
concatenar: nodo 7
crear: tipo 0. valor -474
concatenar: nodo 8
crear: tipo 0. valor -4172
ahijar: nodo 9 en columna 20
crear: tipo 0. valor 3096
crear: tipo 0. valor 2585
crear: tipo 0. valor 4294
crear: tipo 0. valor 4587
crear: tipo 0. valor -4155
crear: tipo 0. valor 3404
concatenar: nodo 15
concatenar: nodo 11
crear: tipo 0. valor -4202
crear: tipo 0. valor -3682
concatenar: nodo 16
crear: tipo 0. valor -1127
ahijar: nodo 12 en columna 5
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
crear: tipo 0. valor 3402
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 14
concatenar: nodo 19
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
concatenar: nodo 10
concatenar: nodo 17
crear: tipo 0. valor 1161
concatenar: nodo 20
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
ahijar: nodo 18 en columna 10
concatenar: nodo 13
crear: tipo 0. valor 3871
crear: tipo 0. valor 1467
ahijar: nodo 22 en columna 11
concatenar: nodo 21
crear: tipo 0. valor 4366
concatenar: nodo 23
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
crear: tipo 0. valor 3084
concatenar: nodo 24
crear: tipo 0. valor -3980
concatenar: nodo 25
crear: tipo 0. valor -537
crear: tipo 0. valor 1281
concatenar: nodo 26
crear: tipo 0. valor -421
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 28
concatenar: nodo 27
crear: tipo 0. valor 1060
crear: tipo 0. valor 4821
crear: tipo 0. valor -2893
ahijar: nodo 31 en columna 24
ahijar: nodo 29 en columna 44
concatenar: nodo 30
crear: tipo 0. valor 3363
crear: tipo 0. valor 2273
concatenar: nodo 33
ahijar: nodo 32 en columna 16
crear: tipo 0. valor 3643
ahijar: nodo 34 en columna 18
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
crear: tipo 0. valor -2222
concatenar: nodo 35
crear: tipo 0. valor 1905
crear: tipo 0. valor -4785
crear: tipo 0. valor 1876
crear: tipo 0. valor 1838
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 36
concatenar: nodo 37
ahijar: nodo 39 en columna 34
ahijar: nodo 38 en columna 47
crear: tipo 0. valor -2835
crear: tipo 0. valor 4063
concatenar: nodo 40
crear: tipo 0. valor -1043
crear: tipo 0. valor -3258
concatenar: nodo 43
crear: tipo 0. valor -3074
crear: tipo 0. valor 406
ahijar: nodo 42 en columna 34
concatenar: nodo 45
concatenar: nodo 44
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
crear: tipo 0. valor -1514
ahijar: nodo 46 en columna 22
crear: tipo 0. valor 76
concatenar: nodo 47
concatenar: nodo 41
crear: tipo 0. valor 3013
concatenar: nodo 48
crear: tipo 0. valor 4096
ahijar: nodo 49 en columna 47
borrar
--- test 3 de 50
crear: tipo 1. valor 73.486076
crear: tipo 1. valor 56.600111
crear: tipo 1. valor 11.070789
concatenar: nodo 2
ahijar: nodo 1 en columna 27
concatenar: nodo 0
crear: tipo 1. valor 66.639717
concatenar: nodo 3
crear: tipo 1. valor 15.344752
ahijar: nodo 4 en columna 31
crear: tipo 1. valor 39.539046
crear: tipo 1. valor -22.976089
crear: tipo 1. valor 39.433572
ahijar: nodo 7 en columna 19
crear: tipo 1. valor 66.194575
concatenar: nodo 6
crear: tipo 1. valor 19.181060
concatenar: nodo 5
concatenar: nodo 8
crear: tipo 1. valor 18.901834
ahijar: nodo 10 en columna 4
crear: tipo 1. valor 17.860445
crear: tipo 1. valor 5.633937
crear: tipo 1. valor 40.431351
crear: tipo 1. valor 49.998269
crear: tipo 1. valor -14.061930
crear: tipo 1. valor 74.383853
concatenar: nodo 15
ahijar: nodo 12 en columna 10
concatenar: nodo 14
concatenar: nodo 16
concatenar: nodo 11
concatenar: nodo 13
imprimir: antes   de filtrar:
filtrar: tipo 1
imprimir: despues de filtrar:
crear: tipo 1. valor -1.946265
concatenar: nodo 17
crear: tipo 1. valor -20.877099
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor 59.286032
crear: tipo 1. valor -20.442493
ahijar: nodo 9 en columna 35
ahijar: nodo 18 en columna 1
concatenar: nodo 19
crear: tipo 1. valor 61.841511
crear: tipo 1. valor -4.192992
crear: tipo 1. valor 51.607588
crear: tipo 1. valor -6.311906
crear: tipo 1. valor 67.578063
crear: tipo 1. valor 25.232501
crear: tipo 1. valor 37.352303
concatenar: nodo 26
crear: tipo 1. valor -10.288176
concatenar: nodo 27
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
concatenar: nodo 23
concatenar: nodo 20
crear: tipo 1. valor 28.280336
concatenar: nodo 21
crear: tipo 1. valor 69.089921
concatenar: nodo 22
concatenar: nodo 28
concatenar: nodo 30
crear: tipo 1. valor 38.562271
concatenar: nodo 24
crear: tipo 1. valor 4.322458
imprimir: antes   de filtrar:
filtrar: tipo 1
imprimir: despues de filtrar:
concatenar: nodo 29
ahijar: nodo 32 en columna 9
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
ahijar: nodo 25 en columna 28
crear: tipo 1. valor 50.428300
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor 57.039474
concatenar: nodo 33
crear: tipo 1. valor -8.504373
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor 3.350786
concatenar: nodo 36
crear: tipo 1. valor 66.516981
crear: tipo 1. valor 40.464311
crear: tipo 1. valor 36.307841
crear: tipo 1. valor -4.331426
crear: tipo 1. valor 60.166916
ahijar: nodo 38 en columna 20
crear: tipo 1. valor 24.658729
ahijar: nodo 39 en columna 26
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
concatenar: nodo 41
concatenar: nodo 34
ahijar: nodo 37 en columna 13
concatenar: nodo 40
crear: tipo 1. valor 57.609020
concatenar: nodo 43
crear: tipo 1. valor -7.767100
crear: tipo 1. valor 59.577376
crear: tipo 1. valor 22.182482
crear: tipo 1. valor 59.650205
ahijar: nodo 42 en columna 45
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
concatenar: nodo 35
concatenar: nodo 45
imprimir: antes   de filtrar:
filtrar: tipo 1
imprimir: despues de filtrar:
crear: tipo 1. valor 30.634272
crear: tipo 1. valor 17.137711
concatenar: nodo 44
concatenar: nodo 31
concatenar: nodo 46
ahijar: nodo 47 en columna 48
concatenar: nodo 48
ahijar: nodo 49 en columna 40
borrar
--- test 4 de 50
crear: tipo 0. valor 514
crear: tipo 0. valor -2655
crear: tipo 0. valor 2513
concatenar: nodo 1
ahijar: nodo 0 en columna 12
concatenar: nodo 2
crear: tipo 0. valor 3056
concatenar: nodo 3
crear: tipo 0. valor -2230
crear: tipo 0. valor -4893
concatenar: nodo 4
crear: tipo 0. valor -402
ahijar: nodo 6 en columna 38
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
crear: tipo 0. valor 1823
crear: tipo 0. valor -3811
crear: tipo 0. valor -3744
crear: tipo 0. valor -1250
concatenar: nodo 7
crear: tipo 0. valor 812
concatenar: nodo 5
crear: tipo 0. valor -4343
crear: tipo 0. valor 1528
crear: tipo 0. valor -1082
crear: tipo 0. valor 2330
ahijar: nodo 13 en columna 14
concatenar: nodo 9
crear: tipo 0. valor 1932
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
crear: tipo 0. valor -1048
concatenar: nodo 14
crear: tipo 0. valor -768
crear: tipo 0. valor -4061
crear: tipo 0. valor -3652
crear: tipo 0. valor -4580
ahijar: nodo 15 en columna 21
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 19
crear: tipo 0. valor 2149
ahijar: nodo 21 en columna 18
concatenar: nodo 17
ahijar: nodo 8 en columna 35
crear: tipo 0. valor 2589
concatenar: nodo 23
ahijar: nodo 20 en columna 46
concatenar: nodo 10
crear: tipo 0. valor -2469
concatenar: nodo 16
crear: tipo 0. valor -2560
crear: tipo 0. valor 2029
ahijar: nodo 22 en columna 40
crear: tipo 0. valor 3172
crear: tipo 0. valor 4643
concatenar: nodo 27
crear: tipo 0. valor -2064
concatenar: nodo 28
concatenar: nodo 11
ahijar: nodo 26 en columna 8
crear: tipo 0. valor 130
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
crear: tipo 0. valor -3110
concatenar: nodo 25
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
concatenar: nodo 12
concatenar: nodo 18
crear: tipo 0. valor -693
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
crear: tipo 0. valor 1407
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 29
concatenar: nodo 30
concatenar: nodo 24
concatenar: nodo 32
crear: tipo 0. valor 377
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
crear: tipo 0. valor -608
crear: tipo 0. valor 1515
concatenar: nodo 33
crear: tipo 0. valor -16
concatenar: nodo 35
crear: tipo 0. valor 4778
concatenar: nodo 36
crear: tipo 0. valor -2592
concatenar: nodo 37
ahijar: nodo 39 en columna 24
ahijar: nodo 38 en columna 0
concatenar: nodo 34
crear: tipo 0. valor -2912
crear: tipo 0. valor 1167
ahijar: nodo 31 en columna 11
ahijar: nodo 40 en columna 37
ahijar: nodo 41 en columna 21
crear: tipo 0. valor 850
concatenar: nodo 42
crear: tipo 0. valor -4204
concatenar: nodo 43
crear: tipo 0. valor 2134
concatenar: nodo 44
crear: tipo 0. valor 1270
concatenar: nodo 45
crear: tipo 0. valor 3543
concatenar: nodo 46
crear: tipo 0. valor 1296
crear: tipo 0. valor 980
concatenar: nodo 48
crear: tipo 0. valor 3146
concatenar: nodo 47
ahijar: nodo 49 en columna 38
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
borrar
