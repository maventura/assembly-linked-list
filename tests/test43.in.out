antes   de colapsar <vacia>despues de colapsar <vacia>antes   de filtrar: { [ -22.164166 ] }despues de filtrar: { [ -22.164166 ] }antes   de colapsar { [ -22.164166 ][ 58.097915 ] }